pkg/sql/colexec/auto_incr.go

```diff
@@ -286,7 +286,7 @@ func getCurrentIndex(param *AutoIncrParam, colName string) (uint64, uint64, erro
 
 func updateAutoIncrTable(param *AutoIncrParam, curNum uint64, name string, mp *mpool.MPool) error {
 	bat := makeAutoIncrBatch(name, curNum, 1, mp)
-	err := param.rel.Delete(param.ctx, bat.GetVector(0), AUTO_INCR_TABLE_COLNAME[0])
```

Comment:
Here 3 columns are passed in and none of the columns are rowid.

Reply:
get it.  I will fix it.

Reply:
@jianwan0214 had fixed it. thanks @jianwan0214